docs/tutorial_xontrib.rst

```diff
@@ -24,15 +24,50 @@ took inspiration from for xonsh:
 * `Sphinx <http://sphinx-doc.org/>`_: Extensions are just Python modules,
   bundles some extensions with the main package, interface is a list of
   string names.
+* `IPython <https://ipython.readthedocs.io/en/stable/config/extensions/index.html>`_: Extensions are just Python modules
+  with some special functions to load/unload.
 * `Oh My Zsh <http://ohmyz.sh/>`_: Centralized registry, autoloading, and
   for a shell.
 * `ESLint <http://eslint.org/>`_: Ability to use language package manager
   to install/remove extensions.
 
-
 Structure
-==========
-Xontribs are modules written in either xonsh (``*.xsh``) or Python (``*.py``).
+================
+Xontribs are modules with some special functions written
+in either xonsh (``*.xsh``) or Python (``*.py``)  that has a couple of special functions to load and unload it.
+
+It is inspired from
```

Comment:
@jnoortheen Did you accidentally delete the rest of this sentence?

Reply:
I think that is the case, there should be ipython,sphinx ... in the mix

Reply:
Ok they are above, I should be removing the hanging sentence